packages/story-editor/src/components/mediaRecording/footer.js

```diff
@@ -231,17 +218,21 @@ function Footer({ captureImage, videoRef }) {
       args.additionalData.isMuted = isMuted;
       args.additionalData.isGif = isGif;
 
-      // Get length from the video blob on screen and fall back
-      // to the duration in state if needed since it's not super reliable.
-      const { length, lengthFormatted } = getVideoLength(videoRef.current);
-      args.resource.length = length || duration;
-      args.resource.lengthFormatted = length
-        ? lengthFormatted
-        : getVideoLengthDisplay(duration);
+      args.resource.length = duration;
```

Comment:
We should round this to a whole number.

Reply:
This is rounded, AFAIK. The original value here comes from the recording counter (which just counts seconds), and [when trimming, the value is updated to the `ceil()` of the trim interval](https://github.com/GoogleForCreators/web-stories-wp/pull/11908/files#diff-cc3a2d938726f362ee058f86a5ce158b8b62d8fa84a3d5abacc96460f980c63aR29-R31).